src/trainers/mitie_trainer.py

```diff
@@ -1,23 +1,26 @@
 from mitie import *
 import os
 import datetime
+import json
 
+from rasa_nlu.featurizers.mitie_featurizer import MITIEFeaturizer
 from rasa_nlu.trainers.trainer import Trainer
 from training_utils import write_training_metadata
 
 
 class MITIETrainer(Trainer):
     SUPPORTED_LANGUAGES = {"en"}
 
-    def __init__(self, config, language_name):
+    def __init__(self, fe_file, language_name):
         self.name = "mitie"
         self.training_data = None
         self.intent_classifier = None
         self.entity_extractor = None
         self.training_data = None
-        self.fe_file = config["fe_file"]
-
+        self.fe_file = fe_file
         self.ensure_language_support(language_name)
+        self.featurizer = MITIEFeaturizer(self.fe_file)
+        del self.featurizer
```

Comment:
What is the purpose of these two lines? Is it to download the fe file from S3? If so I would make that explicitly e.g.
```
if fe_file not present:
   MITIEFeaturizer.download_fe_file(fe_file)
```